app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,288 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  min-height: 1000px;
+  background-color: #f1f3f2 !important;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.groups-holder {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  text-align: center;
+  gap: 2rem;
+}
+
+.groups {
```

Comment:
- [ ] As mentioned by the previous code reviewer, let us try to match the given design. I will display 2 screenshots, first is our project and the other is the required design. Here is a [link](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) to the design

Our App
![Screenshot from 2023-03-22 12-56-43](https://user-images.githubusercontent.com/46388113/227018103-a6435ce0-0485-4c4a-b96e-a89f40a541a6.png)

Required Design
![Screenshot from 2023-03-22 12-56-29](https://user-images.githubusercontent.com/46388113/227018351-c27699c8-c3dd-4844-a723-5f1166dbdc55.png)

Reply:
Fixes design in following commits:
- [d3e7026](https://github.com/mkaya13/budget-app-rails/pull/25/commits/d3e702620b74617aa4c6f01408f3ce93d7079c6a)
- [b1924a1](https://github.com/mkaya13/budget-app-rails/pull/25/commits/b1924a10866cfaeaea413710454cf03180469b5e)

Reply:
- [ ] Great job fixing this as well 👍🏽 
<img width="486" alt="Screenshot 2023-03-22 at 23 22 04" src="https://user-images.githubusercontent.com/58692590/227051709-5b0aab74-21b8-464f-b285-ffef6707a8e3.png">